src/jit/flowgraph.cpp

```diff
@@ -2407,33 +2407,70 @@ BlockSet_ValRet_T   Compiler::fgDomFindStartNodes()
     return startNodes;
 }
 
-/** A simple DFS traversal of the flow graph.
-  * It computes both preorder and postorder numbering.
+/** A non-recursive DFS traversal of the flow graph using an
+  * evaluation stack to assign post-order numbers.
   */
 void Compiler::fgDfsInvPostOrderHelper(BasicBlock* block, BlockSet& visited, unsigned* count)
```

Comment:
This is all new RyuJIT code already, surprised it's not consistent. I see at least 5 different comment styles in this file. Is the // with no extra line the right one?